packages/updating-element/src/updating-element.ts

```diff
@@ -436,13 +449,54 @@ export abstract class UpdatingElement extends HTMLElement {
       ];
       // This for/of is ok because propKeys is an array
       for (const p of propKeys) {
-        // note, use of `any` is due to TypeSript lack of support for symbol in
+        // note, use of `any` is due to TypeScript lack of support for symbol in
         // index types
         // eslint-disable-next-line @typescript-eslint/no-explicit-any
         this.createProperty(p, (props as any)[p]);
       }
     }
     this.elementStyles = this.finalizeStyles(this.styles);
+    // DEV mode warnings
+    if (DEV_MODE) {
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      const warnRemoved = (obj: any, name: string) => {
+        if (obj[name] !== undefined) {
+          console.warn(
+            `\`${name}\` is implemented. It ` +
+              `has been removed from this version of UpdatingElement.`
+          );
+        }
+      };
+      [`render`, `getStyles`].forEach((name: string) =>
```

Comment:
`render` was removed from LitElement, not UpdatingElement, yes?

Reply:
Moved